RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -31,8 +31,10 @@ struct ManageSubscriptionsView: View {
     @StateObject
     private var viewModel: ManageSubscriptionsViewModel
 
-    init(screen: CustomerCenterConfigData.Screen, localization: CustomerCenterConfigData.Localization) {
-        let viewModel = ManageSubscriptionsViewModel(screen: screen, localization: localization)
+    init(screen: CustomerCenterConfigData.Screen,
+    localization: CustomerCenterConfigData.Localization,
+    appearance: CustomerCenterConfigData.Appearance) {
```

Comment:
Indentation seems wrong?

Reply:
Also, I wonder if we should pass the whole config data object, instead of each element separately...